changes-set.txt

```diff
@@ -27,8 +27,25 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 8-Jul-23 bj-ssbcom3lem sbcom3vv merged from BJ's mathbox to main set.mm
```

Comment:
Should all these "merged" say "moved"? I'm not sure what the word "merged" would mean here.

Reply:
It means that the proof already existed, so instead of copy and pasting, the two theorems were "merged" together. I just made this up so if "moved" would still be better then I'll change it

Reply:
Maybe something like "bj-ssbn was an alternative proof of the same theorem"? I'm not sure it is necessary to mention the mathbox part but neither "moved" nor "merged" really explains what happened, at least to my eyes.

Reply:
I'm fine with either, or even not mentioning it in `changes-set.txt`.

Reply:
the wording was revised in https://github.com/metamath/set.mm/pull/3283/commits/c104a28a036c8e3adbac3c93a481ab97ae94e14a

Reply:
New wording looks good to me